build.gradle

```diff
@@ -35,6 +35,8 @@ version = "0.0.0-SNAPSHOT"
 // url to the snapshot repository of a private maven repo.
 def toolkitSnapshotRepo = "http://104.197.230.53:8081/nexus/content/repositories/snapshots/"
 
+def protoVersion = "3.0.0-beta-3"
```

Comment:
nit: I guess this isn't directly related to this PR, since it's also in line 36, but should we stick to single-quoted strings unless the variable replacement feature is needed?